changelog/v1.19.0-patch1/bump-envoy.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
```

Comment:
Noticing that the changelog bot is failing. Should this be released as `1.19.0` instead of `1.19.0-patch1`?

Reply:
i think our release name should be patch1 if envoy already released. to indicate that this is based on a released envoy 1.19.0. the reason we don't just use 1.19.0 is that we may have multiple releases based on 1.19.0

@EItanya - i thought we documented this but i cant find where